Queue the set state requests

samuelsimoes opened the pull request:
With this even when you change your stores many times, only one setState will
be called on the end of the current tick, improving the render performance avoiding
excessive and unnecessary dom diffs calculation on the React component.

Searching for possible bugs...

samuelsimoes committed:
Queue the set state requests

With this even when you change your stores many times, only one setState will
be called on the end of the current tick, improving the render performance avoiding
excessive and unnecessary dom diffs calculation on the React component.

sobrinho commented on dist/fluxo_react_connect_stores.js:
```diff
@@ -57,9 +83,11 @@
           );
 
+        this.stores[storeName] = store;
+
```
@samuelsimoes why keep a reference on `this.stores` since we already have `stores` given to the `FluxoReactConnectStores` function?

samuelsimoes committed:
Adjusting the testing to async code

Now, the view update happens on next tick, so we need use the async
capabilities of mocha.js to test the code.

https://mochajs.org/#asynchronous-code

sobrinho commented:
:shipit:
